src/SeqViewerContainer.tsx

```diff
@@ -84,13 +84,20 @@ class SeqViewerContainer extends React.Component<SeqViewerContainerProps, SeqVie
     this.state = {
       centralIndex: {
         circular: 0,
-        linear: 0,
+        linear: props?.selection?.start || 0,
         setCentralIndex: this.setCentralIndex,
       },
       selection: this.getSelection(defaultSelection, props.selection),
     };
   }
 
+  // If the selection prop updates, also scroll the lineaer view to the new selection
+  componentDidUpdate = (prevProps: SeqViewerContainerProps) => {
+    if (this.props.selection !== prevProps.selection) {
+      this.setCentralIndex("LINEAR", this.props.selection?.start || 0);
+    }
```

Comment:
```suggestion
    if (this.props.selection?.start !== prevProps.selection?.start) {
      this.setCentralIndex("LINEAR", this.props.selection?.start || 0);
    }
```

or something like that? I think we just need to check the start prop and that this is checking the object reference which might change without the start changing? (but I'm a bit out of practice w/ this)

Reply:
Not sure if that could happen, but yeah let's do it just in case. I will also check for the `end`, in case the start remains the same but the end changes